hazelcast/src/main/java/com/hazelcast/map/impl/operation/ContainsKeyOperation.java

```diff
@@ -54,7 +54,7 @@ public Object getResponse() {
 
     @Override
     public WaitNotifyKey getWaitKey() {
-        DistributedObjectNamespace namespace = new DistributedObjectNamespace(MapService.SERVICE_NAME, name);
+        ObjectNamespace namespace = MapService.getObjectNamespace(name);
         return new LockWaitNotifyKey(namespace, dataKey);
```

Comment:
This can be reduced to `return new LockWaitNotifyKey(getServiceNamespace(), dataKey);`.

Reply:
`MapOperation` has `getServiceNamespace()` method already (it seems I forgot to use this before).  Also other `getWaitKey()` methods below can use the same.

Reply:
Done, replaced all invocations in classes extending `MapOperation`